### DISCUSSION TITLE
Package install in the torch enviroment

### DISCUSSION THREAD

**Discussion by LuALu2** [PaddlePaddle/PaddleSpatial#204] [BaiduKDDCup2022] [answered]
Dear Hosts,
Thanks for this exciting competition.
Would it be possible to add 'pytorch geometric' to the torch enviroment?

Thanks.

**Comment by shaido987** [comment, marked as answer]
This is something I requested previously (as part of the requirements for pytorch-geometric-temporal). But there seems to be some issues regarding the cuda version in the running environment, you can see here: https://github.com/PaddlePaddle/PaddleSpatial/discussions/78. In summary: it does not seem possible to install it as of now (although it would be great if you know how the issues mentioned in the link can be solved).

**Reply by LuALu2** [reply]
Thanks for your reply.
I tried below commands in the Colab  (torch version == 1.11.0+cu113):
```
!pip install torch-scatter -f https://data.pyg.org/whl/torch-1.10.0+cu111.html
!pip install torch-sparse -f https://data.pyg.org/whl/torch-1.10.0+cu111.html
!pip install torch-geometric
```
And it seems good. But i m not sure if it will be ok  for the online test env (torch=1.11 & cuda=11.2).

**Reply by shaido987** [reply]
Yes, as you can see from the converstion in the link, I asked the organizers to try with:

```
pip install torch-scatter -f https://data.pyg.org/whl/torch-1.11.0+cu113.html
pip install torch-sparse -f https://data.pyg.org/whl/torch-1.11.0+cu113.html
pip install torch-geometric
```

I.e., using torch 1.11 and cuda 11.3 (since there is no support for 11.2). But it returned some cuda error when running. I'm not sure if cuda 11.1 could work or not though.

**Reply by LuALu2** [reply]
Got it.
Anyway, thanks for your help again.
